.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_SPECIFIC_PLATFORM: "false"
+BUNDLE_WITH: "test"
```

Comment:
I added comments for each of those options, but the disappear after every `bundle install`, so I decided to leave them out to avoid noise in the repo locally. 😞 

The PR description already explains why we have `BUNDLE_SPECIFIC_PLATFORM: "false"`. As for `BUNDLE_WITH: "test"`, my rationale is that it makes local development easier –if you work on this repo you run the tests, right? Right?! 😝– so I left it up to CI to opt-out when appropriate.